### PR TITLE
lightcode

### PR DESCRIPTION
**Pull request by omid-ehsani**

### CODE REVIEW

File: data/lightcode.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "LightCode",
+    "url": "https://lightcode.net/",
+    "career_page_url": "https://lightcode.net/",
+    "type": "Consulting",
+    "categories": [
+       "cloud_software"
+     ],
+     "remote_policy": "Full",
```
Comment (edoardocostantinidev) [inline data/lightcode.json:9]: @omid-ehsani any chance you can point us towards some evidence about full remote employment? Couldn't find any. Even a direct experience is good enough

Comment (edoardocostantinidev) [reply data/lightcode.json:9]: that would be awesome, thanks!

Comment (omid-ehsani) [reply data/lightcode.json:9]: Here are some of our remote workers, they will attest LightCode's full-remote collaboration model:
@DiegoSilvaLightCode @eliodimauro @giovanni-lisitano-lc @luca-lc-colombo @mirko-pozzetti-lightcode @nazarenom @oleg-lc @PasqualeIapalucci @serpratt @umbertocollina-lightcode

Comment (nazarenom) [reply data/lightcode.json:9]: Hello, I'm working full remote from Friuli Venezia Giulia for LightCode

Comment (serpratt) [reply data/lightcode.json:9]: Hi all, I'm working full remote from Calabria for LightCode.

Comment (luca-lc-colombo) [reply data/lightcode.json:9]: Here I am! I do all my work remotely from the province of Milan.

Comment (umbertocollina-lightcode) [reply data/lightcode.json:9]: Hi, I'm working full remote from province of Milan for Lightcode

Comment (oleg-lc) [reply data/lightcode.json:9]: Hi, I'm working full remote for LightCode.

Comment (eliodimauro) [reply data/lightcode.json:9]: Here I am! I do all my work remotely from the province of Latina, Terracina, for Lightcode.

Comment (omid-ehsani) [reply data/lightcode.json:9]: @edoardocostantinidev do you think these referrals satisfy the requirements? Thx.

Comment (edoardocostantinidev) [reply data/lightcode.json:9]: super ! thanks everyone

Comment (giovanni-lisitano-lc) [reply data/lightcode.json:9]: Hi, I'm also are a remote co-worker of LightCode.

**Pull request merged**